src/onegov/org/views/occurrence.py

```diff
@@ -190,6 +190,21 @@ def cors(response):
     ]
 
 
+@OrgApp.view(model=OccurrenceCollection, name='xml', permission=Public)
+def xml_export_events(self, request):
```

Comment:
How will this export been used? This public view is not linked everywhere...

Reply:
The export is used from an mobile app. 
To link it, I can add it to the existing 'export' list http://localhost:8080/onegov_winterthur/winterthur/events/export